.travis.yml

```
@@ -63,10 +63,11 @@ jobs:
     - stage: Lint check
       python: "3.7"
       before_install: # Nothing to do
-      install: pip install flake8 black
+      install: pip install flake8 black isort
       script:
         - flake8 .
         - black --check .
+        - isort -rc .
```

I think it should be like this 
```suggestion
        - isort -rc - c .
```

---

You're right, 8588c603c5accc52a42a0b26547bd94b08f7b017 fixes it.